### PR TITLE
First draft of IPFS Weekly #36

### PR DESCRIPTION
**Pull request by jennwrites**
Looking for feedback and suggestions on this draft so that we can send it out via TinyLetter (and post on the blog) Tuesday, April 2nd. 🌸

Thanks!

### CODE REVIEW

File: published/036-2019-apr-2.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+## Updates and new releases
+*See the latest releases of IPFS tools and projects across the ecosystem.*
+
++ [go-filecoin 0.1.4 is released!](https://filecoin.io/blog/go-filecoin-0.1.4-release/)
+
+
+## Tools and projects we <3
+*[Awesome IPFS](https://awesome.ipfs.io/) is a community maintained and updated list of projects, tools, or pretty much any things related to IPFS that are totally awesome. To see more, or add yours to the list, visit [Awesome IPFS on GitHub](https://github.com/ipfs/awesome-ipfs).* 
+
++ [The Ultimate Guide To Understanding The Differences Between PPIO, Filecoin, and Storj](https://medium.com/@ppio/differences-between-ppio-filecoin-storj-20cdf7b3b02e), 27 Mar 2019
```
Comment (koalalorenzo) [inline published/036-2019-apr-2.md:52]: Another week another example: this is pure advertising to PPIO. Again, does anybody check the articles?

Comment (jennwrites) [reply published/036-2019-apr-2.md:52]: @koalalorenzo I read this article and I was excited to see an IPFS-affiliated project, Filecoin, being mentioned. 

I apologize for your distaste with the newsletter, but it's a free community resource and no one is forcing you to read it. I would be happy to see any of your suggestions for more suitable content for this week, if you have any.

Comment (koalalorenzo) [reply published/036-2019-apr-2.md:52]: I should apologise, I didn't know you were sharing this kind of content too. I am also sorry that I don't have any suitable content this week. When I will have I will be sure to share it with you! 👍

**Comment by jennwrites** [comment]
IPFS Weekly #36 was published:

On the [IPFS blog](https://blog.ipfs.io/77-ipfs-weekly-36/) 
Sent to 2943 subscribers [via Tinyletter](https://tinyletter.com/ipfsnewsletter/letters/ipfs-weekly-36)

**Pull request merged**